src/Digdir.Domain.Dialogporten.Janitor/Program.cs

```diff
@@ -61,6 +67,8 @@ static void BuildAndRun(string[] args)
 
     var app = builder.Build();
 
+    Log.Information("Current environment: {Environment}", app.Environment.EnvironmentName);
```

Comment:
Er denne lagt inn for debug purposes, eller tenker vi at denne skal stå?

Reply:
Tenker at den egentlig er overflødig ettersom vi skal finne den verdien i DOTNET_ENVIRONMENT miljø variabelen. Hva tenker du  @arealmaas? 🙂

Reply:
Enig enig!